pxf/Makefile

```diff
@@ -42,6 +42,10 @@ ifneq "$(PXF_VERSION)" ""
     BUILD_PARAMS+= -Pversion="$(PXF_VERSION)"
 endif
 
+ifneq "$(GRADLEW_OPTIONS)" ""
```

Comment:
The purpose of Makefile is to abstract the user from gradle options/commands. Should we instead call this BUILD_OPTIONS ?